search-service/src/main/kotlin/com/egm/stellio/search/service/AttributeInstanceService.kt

```diff
@@ -86,8 +90,19 @@ class AttributeInstanceService(
             .fetch()
             .all()
             .map {
-                it.plus(Pair("attribute_name", temporalEntityAttribute.attributeName))
+                rowToAttributeInstanceResult(it, temporalEntityAttribute)
             }
             .collectList()
     }
+
+    private fun rowToAttributeInstanceResult(row: Map<String, Any>, temporalEntityAttribute: TemporalEntityAttribute): AttributeInstanceResult {
```

Comment:
it seems to be longer than 120 characters

Reply:
indeed